admin_guide/managing_networking.adoc

```diff
@@ -1322,3 +1322,39 @@ such as Google can construct a list of sites that have `preload` set. Browsers
 can then use these lists to determine which sites to only talk to over HTTPS,
 even before they have interacted with the site. Without `preload` set, they need
 to have talked to the site over HTTPS to get the header.
+
+[[admin-guide-enabling-iperf]]
+== Troubleshooting Throughput Issues
+
+There are situations where the applications deployed through {product-title} can cause 
+network throughput issues such as unusually high latency between specific services. 
+
+Use the following steps to analyze performance issues if pod logs do not reveal any cause of the problem:
+
+* Use a packet analyzer, such as link:http://www.tcpdump.org/[tcpdump] to analyze traffic between a pod and its node.
++
+For example, you could run tcpdump on each pod while reproducing the behavior that led to the issue.
++
+----
+$ tcpdump -s 0 -i any -w /tmp/dump.pcap host <podip 1> && host <podip 2> <1>
+----
++
+<1> `podip` is the IP address for the pod. Run the following to get the IP address of the pods:
++
+----
+# oc get pod <podname> -o wide 
+----
++
+tcpdump generates a file at /tmp/dump.pcap containing all traffic between these two pods. Ideally, run the analyzer shortly 
+before the issue is reproduced and stop the analyzer shortly after the issue is finished reproducing to minimize the size of the file. 
+You can also run a packet analyzer between the nodes (eliminating the SDN from the equation) with: 
++
+----
+# tcpdump -s 0 -i any -w /tmp/dump.pcap port 4789
+----
+
+* Check whether the node interface overloaded with traffic from other pods, storage devices, or the data plane?
+
+* Use a tool such as  link:https://iperf.fr/[iperf] to measure streaming throughput and UDP throughput. Run the tool from the pods first, then from the nodes
```

Comment:
you might want to note here that iperf3 is included in RHEL7 (rather than linking to the website).